### PR TITLE
Rename quadratic matrix into square matrix

### PR DESCRIPTION
**Pull request by danielrademacher**
Some progress of Issue #136.

### CODE REVIEW

File: experimental/Matrix/matrix.jl [hunk 1]
```diff
@@ -135,6 +135,26 @@ function _lex_isEqual(a::T,b::T) where T<:MatElem
   return true
 end
 
+function size(G::GapObj)
```
Comment (fingolfin) [inline experimental/Matrix/matrix.jl]: This function should go! Use order instead, which already exists

File: experimental/Matrix/matrix.jl [hunk 2]
```diff
@@ -135,6 +135,26 @@ function _lex_isEqual(a::T,b::T) where T<:MatElem
...
+
+function maximalsubgroups(G::GapObj)
+    return GAP.Globals.MaximalSubgroups(G)
+end
+
+function smallgeneratingset(G::GapObj)
+    return GAP.Globals.SmallGeneratingSet(G)
+end
+
+function isomorphismpermgroup(G::GapObj)
```
Comment (fingolfin) [inline experimental/Matrix/matrix.jl]: Please take a look at the oscar manual, many of these functions already exist, possibly under slightly different names

File: experimental/Matrix/matrix.jl [hunk 3]
```diff
@@ -67,7 +67,7 @@ function MatrixGroup(matrices::Vector{<:MatrixElem{T}}) where T <: Union{fmpz, f
             if !isinvertible(mat)
                 error("At least one matrix is not invertible.")
             end
-            if size(mat) != (n, n)
+            if Oscar.size(mat) != (n, n)
                 error("At least one matrix is not quadratic or not the same size.")
```
Comment (joschmitt) [inline experimental/Matrix/matrix.jl]: I've been told that it is called "square matrix" and not "quadratic matrix" in English.

Comment (fingolfin) [reply experimental/Matrix/matrix.jl]: Indeed. Also note that the function `issquare` can be used for this purpose.

File: experimental/Matrix/matrix.jl [hunk 4]
```diff
@@ -67,8 +67,8 @@ function MatrixGroup(matrices::Vector{<:MatrixElem{T}}) where T <: Union{fmpz, f
             if !isinvertible(mat)
                 error("At least one matrix is not invertible.")
             end
-            if size(mat) != (n, n)
-                error("At least one matrix is not quadratic or not the same size.")
+            if Oscar.size(mat) != (n, n)
```
Comment (fingolfin) [inline experimental/Matrix/matrix.jl]:
```suggestion
            if size(mat) != (n, n)
```

**Review by fingolfin — APPROVED** [review]

**Pull request merged**